plugins/laser/Laser.cc

```diff
@@ -1,4 +1,6 @@
 #include "LaserDriver.cpp"
+
+#include <gz/msgs/details/laserscan.pb.h>
```

Comment:
Is this intentional?

Reply:
Yes without it I got a compilation error, do you think is it the wrong include?

Reply:
The fact that it is a detail header suggests that this is a private header and should not be included. Anyhow, this is not blocking

Reply:
Thank you for the suggestion, in https://github.com/robotology/gz-sim-yarp-plugins/pull/105/commits/efa5a07771e77b889d0b92cd8c670f550cd8093e I updated it to use the main msgs header, as suggested in Gazebo documentation: https://github.com/robotology/gz-sim-yarp-plugins/pull/105/commits/efa5a07771e77b889d0b92cd8c670f550cd8093e